demo/server/src/modules/keycloak/index.ts

```diff
@@ -0,0 +1,36 @@
+/// <reference path="./keycloak.d.ts" />
+
+import * as express from 'express';
+import * as session from 'express-session';
+import Keycloak = require('keycloak-connect');
+
+// Express Session Configuration.
+const sessionOpts = {
```

Comment:
[Idea] Session options seem to be common for both passport and keycloak. Maybe we can share the same options in code to make sure that we will avoid differences and possible security issues in outdated ones. ping @JameelB for opinion.